extensions/ql-vscode/src/view/data-extensions-editor/DataExtensionsEditor.tsx

```diff
@@ -252,6 +254,10 @@ export function DataExtensionsEditor({
     });
   }, [viewState?.mode]);
 
+  const onHideModeledApis = useCallback(() => {
```

Comment:
To be picky (sorry I didn't mention this while pairing) and to fit in with conventions, I believe we should call this `handleHideModeledApisChanged` or something like that. Calling the function itself `handleX` is apparently the standard convention, especially when created using `useCallback`.

Reply:
Thanks! [As discussed offline, I'll leave the name as-is for now since the rest of this file also doesn't follow the `handleXXX` convention 😅 ]